python/cugraph/cugraph/tests/dask/test_mg_bfs.py

```diff
@@ -85,6 +85,70 @@ def modify_dataset(df):
     assert err == 0
 
 
+@pytest.mark.skipif(
+    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+)
+def test_dask_bfs_undirected(dask_client):
+    gc.collect()
```

Comment:
I believe we are no longer calling `gc` this way (unfortunately we still have few of them in our code)

Reply:
Done..moved it up to a setup function at the module level per Rick.